app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -110,14 +110,15 @@ export class ExecutionGroup extends React.Component<IExecutionGroupProps, IExecu
       });
   }
 
-  public triggerPipeline(): void {
+  public triggerPipeline(trigger: any = null): void {
```

Comment:
Is this not an `ITrigger`?

Reply:
I don't know, it's a null and TS was complaining about the implicit any so I really wanted to stick it to the compiler but from your reaction that isn't the right way to fix this so I'll cast it to an `ITrigger`